### PR TITLE
[TTAHUB-789] Fix Filter Menu Resize when Removing Last Item

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Removing the last "applied" filter item causes the filter menu to squish.

## How to test

Apply a filter item to any page using the new filters. When removing this item the filter menu should retain its original size. Closing and opening the filter menu should still have a new blank filter item.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-789

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/components/filter/FilterMenu.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import FilterItem from './FilterItem';
 import usePrevious from '../../hooks/usePrevious';
 import { filterProp, filterConfigProp } from './props';
 import FilterErrorContext from './FilterErrorContext';
+import './filterMenu.css';
```
Comment (thewatermethod) [inline frontend/src/components/filter/FilterMenu.js:13]: _incredibly_ small thing but we usually capitalize to match react component

**Review by thewatermethod — APPROVED** [review]
Thanks for organizing, fix makes sense & works
I would just double check at standup and see if this fix satisfactorily closes the ticket

File: frontend/src/components/filter/FilterMenu.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/components/filter/FilterMenu.js:13]: Name has been updated.

**Review by jasalisbury — APPROVED** [review]
:shipit:

**Pull request merged**